cg/constants/pacbio.py

```diff
@@ -74,13 +85,35 @@ class PacBioBundleTypes:
     SMRT_CELL: str = "smrt_cell"
 
 
+class BarcodeMetricsAliases:
+    BARCODED_HIFI_READS: str = "barcode.n_barcoded_reads"
+    BARCODED_HIFI_READS_PERCENTAGE: str = "barcode.percent_barcoded_reads"
+    BARCODED_HIFI_YIELD: str = "barcode.barcoded_bases"
+    BARCODED_HIFI_YIELD_PERCENTAGE: str = "barcode.percent_barcoded_bases"
+    BARCODED_MEAN_READ_LENGTH: str = "barcode.n_barcoded_readlength_mean"
+    UNBARCODED_HIFI_READS: str = "barcode.n_unbarcoded_reads"
+    UNBARCODED_HIFI_YIELD: str = "barcode.unbarcoded_bases"
+    UNBARCODED_HIFI_MEAN_READ_LENGTH: str = "barcode.mean_unbarcoded_read_length"
+
+
+class SampleMetricsAliases:
+    BARCODE_NAME: str = "barcode.barcode_table.barcode"
+    HIFI_MEAN_READ_LENGTH: str = "barcode.barcode_table.mean_read_length"
+    HIFI_READ_QUALITY: str = "barcode.barcode_table.median_read_quality"
+    HIFI_READS: str = "barcode.barcode_table.number_of_reads"
+    HIFI_YIELD: str = "barcode.barcode_table.number_of_bases"
+    PLOYMERASE_READ_LENGTH: str = "barcode.barcode_table.mean_polymerase_read_length"
+    SAMPLE_NAME: str = "barcode.barcode_table.biosample"
```

Comment:
I think NAME is misleading in our context. Shoul we go for INTERNAL_ID or just ID?
```suggestion
    SAMPLE_INTERNAL_ID: str = "barcode.barcode_table.biosample"
```

Reply:
I think internal id makes sense

Reply:
fixed